### PR TITLE
Mention alternatives to deprecated functions in CHANGELOG v1.4

### PR DESCRIPTION
**Pull request by eksperimental**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -272,21 +272,21 @@ It is also possible to install escripts and archives by providing a Git/GitHub r
   * [Access] `Access.key/1` is deprecated due to erratic behaviour for missing keys, please use `Access.key/2` instead with proper default values
   * [Behaviour] The `Behaviour` module is deprecated. Callbacks may now be defined directly via the `@callback` attribute
   * [Enum] Deprecate `Enum.uniq/2` in favor of `Enum.uniq_by/2`
-  * [Float] `Float.to_char_list/2` and `Float.to_string/2` are deprecated (use the :erlang functions if such conversions are desired)
+  * [Float] `Float.to_char_list/2` and `Float.to_string/2` are deprecated (use the `:erlang.float_to_list/2` and `:erlang.float_to_binary/2` functions if such conversions are desired)
```
Comment (lexmag) [inline CHANGELOG.md]: I'm wondering what that strange whitespace change here?

File: CHANGELOG.md [hunk 2]
```diff
@@ -272,21 +272,21 @@ It is also possible to install escripts and archives by providing a Git/GitHub r
...
   * [Kernel] Deprecate support for making private functions overridable. Overridable functions must always be public as they must be contracts
   * [Kernel] Warn if variable is used as a function call
   * [OptionParser] Deprecate aliases with multiple letters, such as `-abc`
   * [Set] Deprecate the `Set` module
-  * [Stream] Deprecate `Stream.uniq/2` in favor of `Stream.uniq_by/2`
+  * [Stream] Deprecate `Stream.uniq/2`
```
Comment (lexmag) [inline CHANGELOG.md]: I assume this change is accidental?

File: CHANGELOG.md (hunk 1 above)
Comment (eksperimental) [inline CHANGELOG.md]: I'm editing directly on the web interface, who know what this thing is doing...

**Comment by eksperimental** [comment]
@lexmag I have removed the weird added whitespaces, and fixed everything. let me know what you think

**Comment by lexmag** [comment]
Nice! Thanks @eksperimental.

**Pull request merged**